js/piemenus.js

```diff
@@ -605,7 +596,10 @@ const piemenuPitches = (
             that.value += selection["attr"];
             that.text.text = selection["note"] + selection["attr"];
         }
-
+        // Store the selected accidental in the block for later use.
+        prevAccidental = selection["attr"];
+        block.prevAccidental = prevAccidental;
```

Comment:
This retains the value during the session, but not between sessions. Maybe we need to think about saving/restoring this attribute with the block (much the way we do with the start block attributes).

Reply:
Like do you want the pitch block to save the data for last session like this @walterbender

Reply:
So what should I change in this pr regarding the issue @walterbender

Reply:
I think you don't need to change anything in what you have already done, but in this function in activity.js, `this.prepareExport`, you can see how some attributes are stored in the `start` block. We'll want to do something similar with the pitch block. And then restore the state on project import (in `loadNewBlocks` in `blocks.js`.

But maybe all that complexity can be dealt with in a separate PR. Let me test this code as is.

Reply:
Okay @walterbender i ll make a new PR in regards to these changes and will resolve soon.